### PR TITLE
Update MS-ASAIRS (v20190618) RS/Configuration script / MD

### PR DESCRIPTION
**Pull request by FangfangFan**
1. Update RS  for  MS-ASAIRS (v20190618) 
2. Configuration script for exchange 2019 
3. MD file

### CODE REVIEW

File: ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml [hunk 1]
```diff
@@ -2499,6 +2752,15 @@ If the client requests a text body whose native format is HTML, and the size of
...
 		<ns1:Verification>Unverified</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASAIRS_R1000965</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.14</ns1:Doc_Sect>
+		<ns1:Description>[In ClientId] Protocol version 16.1 supports this [ClientId] element.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml:2762]: Should be Adapter

File: ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml [hunk 2]
```diff
@@ -3021,10 +3337,19 @@ If the client requests a text body whose native format is HTML, and the size of
...
 		<ns1:Verification>Unverified</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASAIRS_R1000977</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.18.2</ns1:Doc_Sect>
+		<ns1:Description>[In ContentType (Properties)] Protocol version 16.1 supports this [ContentType (Properties)] element.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml:3347]: adapter

File: ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml [hunk 3]
```diff
@@ -3908,7 +4332,16 @@ If the client requests a text body whose native format is HTML, and the size of
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-ASAIRS_R1368</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.2.24.1</ns1:Doc_Sect>
-		<ns1:Description>[In FileReference (Attachment)] In protocol version 16.0, the server will return the FileReference element along with the ClientId element (section 2.2.2.14) as child elements of the Attachment element in response to a Sync command request ([MS-ASCMD] section 2.2.2.20) that adds an attachment [either to a calendar item or] to a draft email item. </ns1:Description>
+		<ns1:Description>In protocol version 16.0, the server will return the FileReference element along with the ClientId element (section 2.2.2.14) as child elements of the Attachment element in response to a Sync command request ([MS-ASCMD] section 2.2.1.21) that adds an attachment either to a calendar item or to a draft email item. </ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml:4339]: adapter

File: ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml (hunk 1 above)
Comment (FangfangFan) [reply ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml:2762]: The similar RS below marked as Unverified in last RS update.

MS-ASAIRS_R1086 | 2.2.2.14 | [In ClientId]   Protocol version 16.0 supports this [ClientId] element. |   | Protocol | Both | Normative | Unverified
-- | -- | -- | -- | -- | -- | -- | --

File: ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml (hunk 2 above)
Comment (FangfangFan) [reply ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml:3347]: The similar RS below marked as Unverified in last RS update.

MS-ASAIRS_R1123 | 2.2.2.18.2 | [In ContentType   (Properties)] Protocol version 16.0 supports this [ContentType (Properties)]   element. |   | Protocol | Both | Normative | Unverified
-- | -- | -- | -- | -- | -- | -- | --

**Review by XinwLi — APPROVED** [review]

File: ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml (hunk 3 above)
Comment (FangfangFan) [reply ExchangeActiveSync/Source/MS-ASAIRS/TestSuite/MS-ASAIRS_RequirementSpecification.xml:4339]: The similar RS below marked as Unverified in last RS update.
[In   FileReference (Attachment)] In protocol version 16.0, the server will return   the FileReference element along with the ClientId element (section 2.2.2.14)   as child elements of the Attachment element in response to a Sync command   request ([MS-ASCMD] section 2.2.2.20) that adds an attachment [either] to a   calendar item [or to a draft email item]. |   | Protocol | Both | Normative | Unverified
-- | -- | -- | -- | -- | --

**Pull request merged**